configuration/configuration.go

```diff
@@ -68,6 +68,10 @@ const (
 	varPostgresConnectionRetrySleep = "postgres.connection.retrysleep"
 	varHTTPAddress                  = "http.address"
 	varDeveloperModeEnabled         = "developer.mode.enabled"
+	varGithubSecret                 = "github.secret"
+	varGithubClientID               = "github.client.id"
+	varTokenPublicKey               = "rsa.public.key"
+	varTokenPrivateKey              = "rsa.private.key"
```

Comment:
The yaml keys in the configs (`README.adoc` and `config.yaml`) is `token.privatekey` and `token.publickey`. If you don't change the `rsa.client.id` and `rsa.public.key`, those values will never be picked up and only the defaults will be used.